app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/utils/CheckYourAnswersHelper.scala

```diff
@@ -626,7 +626,7 @@ class CheckYourAnswersHelper @Inject() (implicit langs: Langs, messages: Message
           Actions(
             items = Seq(
               ActionItem(
-                href = s"${fileUploadRoutes.SupportingEvidenceController.checkYourAnswers().url}",
+                href = fileUploadRoutes.SupportingEvidenceController.checkYourAnswers(isAmend = true).url,
```

Comment:
do you think this would be better implemented if we had a separate endpoint to handle this? for example `fileUploadRoutes.SupportingEvidenceController.changeYourAnswers().url` ?

Reply:
It is all yak, can add separate endpoint not a problem